go/integTest/standalone_commands_test.go

```diff
@@ -1017,3 +1017,139 @@ func (suite *GlideTestSuite) TestFunctionStats() {
 		assert.Equal(suite.T(), int64(0), nodeStats.Engines["LUA"].FunctionCount)
 	}
 }
+
+// Helper method to skip tests on versions lower than specified
+func (suite *GlideTestSuite) skipOnVersionLowerThan(version, reason string) {
```

Comment:
We have that in `glide_test_suite_test.go` : https://github.com/valkey-io/valkey-glide/blob/8febf8a2090d4e7d3daa5adb6110432bc6a68f18/go/integTest/glide_test_suite_test.go#L380-L384